packages/angular/package.json

```diff
@@ -19,7 +19,7 @@
     "@angular/platform-browser-dynamic": "^16.2.0",
     "@angular/router": "^16.2.0",
     "@flatfile/api": "^1.6.7",
-    "@flatfile/embedded-utils": "^1.1.7",
+    "@flatfile/embedded-utils": "^1.1.3",
```

Comment:
Why drop the version back?

Reply:
Well, 1.1.7 was a typo from somewhere so this just fixes that.